hevm.cabal

```diff
@@ -130,35 +112,35 @@ library
     Decimal                           >= 0.5.1 && < 0.6,
     containers                        >= 0.6.0 && < 0.7,
     deepseq                           >= 1.4.4 && < 1.5,
-    time                              >= 1.8.0 && < 1.11,
+    time,
```

Comment:
Can we add version bounds to all the deps here. Hackage are kinda fussy about it.